Startup.cs

```diff
@@ -44,7 +57,17 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
 
             app.UseEndpoints(endpoints =>
             {
-                endpoints.MapControllers();
+                endpoints.MapControllerRoute(
+                    name: "default",
+                    pattern: "{controller=Home}/{action=Index}/{id?}");
```

Comment:
i think this is mvc and not web API 
use map controllers 

 app.UseEndpoints(endpoints => endpoints.MapControllers());